### DISCUSSION TITLE
How do I turn on subtitles and switch between non-English languages?

### DISCUSSION THREAD

**Discussion by SinVllMocha** [sultim-t/xash-rt#195] [Q&A]
How do I turn on subtitles and switch between non-English languages?

**Comment by stephen-cusi** [comment]
Unfortunately, there are no subtitles

**Reply by SinVllMocha** [reply]
oh nvm,I just realized that the game doesn't have subtitles, but that brings me to the second question, can I switch languages? This cannot be switched directly in steam

**Reply by SinVllMocha** [reply]
Ah, I can just replace the language file, and the matter is resolved

**Reply by stephen-cusi** [reply]
Overwrite the original sound folder with other language versions

**Reply by l4ucha** [reply]
Hi! How did you changed the language file for the mod? And where did you get the language file? I want to change it to spanish. TY!

**Reply by Kjllvibes1** [reply]
> Hi! How did you changed the language file for the mod? And where did you get the language file? I want to change it to spanish. TY!

have you been able to find any fix? Im italian and im trying to play the game in that language but i can only play it in english
